HousingRepairsOnlineApi/Gateways/AzureStorageGateway.cs

```diff
@@ -28,5 +29,40 @@ public async Task<string> UploadBlob(string base64Img, string fileExtension)
 
             return blobClient.Uri.AbsoluteUri;
         }
+
+        public async Task<string> GetServiceSasUriForBlob(string fileName, int daysUntilExpiry, string storedPolicyName = null)
+        {
+
+            var blobClient = storageContainerClient.GetBlobClient(fileName);
+
+            if (blobClient.CanGenerateSasUri)
+            {
+                var sasBuilder = new BlobSasBuilder
+                {
+                    BlobContainerName = storageContainerClient.Name,
+                    BlobName = blobClient.Name,
+                    Resource = "b"
```

Comment:
Refactor this magic string to a class constant with a more descriptive name, i.e. `BlobResourceType`